src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,257 @@
+#Contract Entities
+####AuditableEntity
+#####What is it?
+`AuditableEntity` is the abstract base type for all OpenHDS entities. The class provides fields useful for ‘auditing’ or keeping track of meta information pertaining to the concrete type like when it was created and last modified.
+######Required Fields:
++ String uuid 
++ User insertBy 
++ Date insertDate 
++ User lastModifiedBy 
++ Date lastModifiedDate 
+
+___
+####AuditableCollectedEntity
+(child of `AuditableEntity`)
+#####What is it?
+`AuditableCollectedEntity` is the abstract base type for all OpenHDS entities that are collected by a `FieldWorker`.
+######Dependencies
++ FieldWorker
+
+######Required Fields:
++ FieldWorker collectedBy 
++ Date collectionDateTime
+
+___
+####AuditableExtIdEntity
+(child of `AuditableCollectedEntity`) 
+#####What is it?
+`AuditableExtIdEntity` is an entity that is identifiable by an id external to the OpenHDS, i.e. the functionality of the OpenHDS is not dependent on it. 
+######Required Fields:
++ String extId 
+
+#Census Entities
+####LocationHierarchy
+(child of `AuditableExtIdEntity`)
+#####What is it?
+`LocationHierarchy` is a node in the tree that represents the study area. For example, at the top of a tree would be the The United States and it would have a child node for each of the 50 states. Each state node could have a child node for its cities and each city could have a child node for its districts and so on. `LocationHierarchy` extends `AuditableExtIdEntity` meaning it is identified by uuid as well as extId. 

```

Comment:
I think it would be better to list abstract fields like uuid explicitly with each entity.  If the reader wants to know all the fields required for LocationHierarchy, I think they should be able to just read them, and not have to assemble them from different locations.